tests/testthat/test-dataframe.R

```diff
@@ -927,3 +927,23 @@ test_that("rename", {
   b = df$rename(list(miles_per_gallon = "mpg", horsepower = "hp"))$columns
   expect_identical(a, b)
 })
+
+
+
+test_that("describe", {
+  expect_snapshot(pl$DataFrame(mtcars)$describe()$to_list(),variant = "deparse")
```

Comment:
why do you use `to_list()`? I think the snapshot is easier to read with the default output:
```r
> pl$DataFrame(mtcars)$describe()
shape: (9, 12)
┌────────────┬───────────┬──────────┬────────────┬───┬──────────┬──────────┬──────────┬────────┐
│ describe   ┆ mpg       ┆ cyl      ┆ disp       ┆ … ┆ vs       ┆ am       ┆ gear     ┆ carb   │
│ ---        ┆ ---       ┆ ---      ┆ ---        ┆   ┆ ---      ┆ ---      ┆ ---      ┆ ---    │
│ str        ┆ f64       ┆ f64      ┆ f64        ┆   ┆ f64      ┆ f64      ┆ f64      ┆ f64    │
╞════════════╪═══════════╪══════════╪════════════╪═══╪══════════╪══════════╪══════════╪════════╡
│ count      ┆ 32.0      ┆ 32.0     ┆ 32.0       ┆ … ┆ 32.0     ┆ 32.0     ┆ 32.0     ┆ 32.0   │
│ null_count ┆ 0.0       ┆ 0.0      ┆ 0.0        ┆ … ┆ 0.0      ┆ 0.0      ┆ 0.0      ┆ 0.0    │
│ mean       ┆ 20.090625 ┆ 6.1875   ┆ 230.721875 ┆ … ┆ 0.4375   ┆ 0.40625  ┆ 3.6875   ┆ 2.8125 │
│ std        ┆ 6.026948  ┆ 1.785922 ┆ 123.938694 ┆ … ┆ 0.504016 ┆ 0.498991 ┆ 0.737804 ┆ 1.6152 │
│ min        ┆ 10.4      ┆ 4.0      ┆ 71.1       ┆ … ┆ 0.0      ┆ 0.0      ┆ 3.0      ┆ 1.0    │
│ max        ┆ 33.9      ┆ 8.0      ┆ 472.0      ┆ … ┆ 1.0      ┆ 1.0      ┆ 5.0      ┆ 8.0    │
│ median     ┆ 19.2      ┆ 6.0      ┆ 196.3      ┆ … ┆ 0.0      ┆ 0.0      ┆ 4.0      ┆ 2.0    │
│ 25pct      ┆ 15.5      ┆ 4.0      ┆ 121.0      ┆ … ┆ 0.0      ┆ 0.0      ┆ 3.0      ┆ 2.0    │
│ 75pct      ┆ 22.8      ┆ 8.0      ┆ 350.0      ┆ … ┆ 1.0      ┆ 1.0      ┆ 4.0      ┆ 4.0    │
└────────────┴───────────┴──────────┴────────────┴───┴──────────┴──────────┴──────────┴────────┘
```

Reply:
I got this error with that the "roundtrip" conversion failed. I guess it was because I used `expect_snapshot_value` which does support polars objects. I will take your advice and use expect_snapshot and capture the printed form instead :)